gu-hub/src/main.rs

```diff
@@ -3,6 +3,7 @@ extern crate gu_actix;
 extern crate gu_base;
 extern crate gu_event_bus;
 extern crate gu_hardware;
+extern crate gu_hdman;
```

Comment:
I don't like import of gu_hdman in hub crate. hdman is nearly exclusively related to the provider.  I think moving out DownloadOptionsBuilder from the gu_hdman crate is the best option.

Reply:
Any other comments?

Reply:
@filipgolem pls respond to this comment

Reply:
Yes, it's a good idea to move the downloader to a separate crate. This is not related to this pull request; the plugin installer uses the downloader.